packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -193,6 +210,27 @@ class BottomSheetRangeCell extends Component {
 				}
 			>
 				<View style={ styles.container }>
+					{ columnsPreview && (
```

Comment:
I think that it shouldn't be a part of range-cell since it is very specific to the columns block. Could you maybe create a new component where you use range-control and columns preview together instead of adding it here?